rest-api-spec/src/main/resources/rest-api-spec/api/indices.exists_template.json

```diff
@@ -8,15 +8,11 @@
       "parts": {
         "name": {
           "type": "list",
-          "required": false,
+          "required": true,
           "description": "The comma separated names of the index templates"
         }
       },
       "params": {
-        "flat_settings": {
```

Comment:
`flat_settings` is supported on `HEAD` requests; the behavior of `GET` and `HEAD` are aligned when both methods are supported.